src/data/ons_pd_may_19.py

```diff
@@ -45,58 +47,64 @@ def __init__(self, ons_pd_csv_path, load_data=True):
         ONSPostcodeDirectory.__init__(self, ons_pd_csv_path, load_data, ONSPostcodeDirectoryMay19.index_column, ONSPostcodeDirectoryMay19.fields, ONSPostcodeDirectoryMay19.data_types)
 
         # Folder within the ONS Postcode Directory archive holding names and codes files
-        self.NAMES_AND_CODES_FILE_LOCATION = self.settings["ONS Names and codes folder"]
+        NAMES_AND_CODES_FOLDER = self.settings["ONS Names and codes folder"]
```

Comment:
I think I prefer this lowercase but am not sure ... maybe someway of denoting the variable is from 'settings' is useful though ...

Reply:
done